Makefile

```diff
@@ -0,0 +1,37 @@
+# TODO: this Makefile will probably be reusable, we should consider moving it somewhere
```

Comment:
Heavy +1

Reply:
I think that we could move it to infra, at least for now.

Reply:
@se7entyse7en infra is private, create a CI repo maybe?

Reply:
oh, right. Yup, a CI repo makes sense IMO.

Reply:
Done